### PR TITLE
Update Arduino.cc.xml

### PR DESCRIPTION
**Pull request by Rotzbua**
more domains offer https

### CODE REVIEW

File: src/chrome/content/rules/Arduino.cc.xml [hunk 1]
```diff
@@ -1,26 +1,20 @@
...
 <ruleset name="Arduino.cc (partial)">
-
+	<target host="arduino.cc" />
+	<target host="www.arduino.cc" />
+	<target host="blog.arduino.cc" />
+	<target host="cdn.arduino.cc" />
 	<target host="id.arduino.cc" />
-
+	<target host="playground.arduino.cc" />
+	<target host="store.arduino.cc" />
```
Comment (J0WI) [inline src/chrome/content/rules/Arduino.cc.xml]: See also:
```
auth.arduino.cc
casajasmina.arduino.cc (refused)
create.arduino.cc
cloud.arduino.cc
day.arduino.cc
downloads.arduino.cc
forum.arduino.cc

Comment (Rotzbua) [reply src/chrome/content/rules/Arduino.cc.xml]: thanks, sub domains added

**Comment by J0WI** [comment]
`ERROR src/chrome/content/rules/Arduino.cc.xml: Fetch error: http://auth.arduino.cc/ => https://auth.arduino.cc/: need more than 1 value to unpack`

@Hainish we have some more PRs with this error. Do you have any clue what this means?

**Comment by Rotzbua** [comment]
@J0WI @Hainish The reason seems to be a broken http response. There is a redirect but the `location` is wrong.
```HTTP Headers for http://auth.arduino.cc

HTTP/1.1 301 Moved Permanently
Date: Sat, 11 Nov 2017 18:13:40 GMT
Content-Type: text/html
Content-Length: 185
Connection: keep-alive
Set-Cookie: AWSALExpires=Sat, 18 Nov 2017 18:13:40 GMT; Path=/
Server: nginx/1.13.6
Location: https:///

**Pull request merged**